app/src/main/java/com/x8bit/bitwarden/ui/platform/manager/biometrics/BiometricsManagerImpl.kt

```diff
@@ -50,14 +50,21 @@ class BiometricsManagerImpl(
         onCancel: () -> Unit,
         onLockOut: () -> Unit,
         onError: () -> Unit,
+        onNotSupported: () -> Unit,
     ) {
-        configureAndDisplayPrompt(
-            onSuccess = { onSuccess() },
-            onCancel = onCancel,
-            onLockOut = onLockOut,
-            onError = onError,
-            cipher = null,
-        )
+        when {
+            isUserVerificationSupported.not() -> onNotSupported()
```

Comment:
Can this just be an `if`?

Or will more cases be coming?

Reply:
It can be. I don't expect any more cases to be added.